presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/DisabledGlueColumnStatisticsProvider.java

```diff
@@ -51,15 +49,15 @@
     }
 
     @Override
-    public void updateTableColumnStatistics(TableInput table, Map<String, HiveColumnStatistics> columnStatistics)
+    public void updateTableColumnStatistics(Table table, Map<String, HiveColumnStatistics> columnStatistics)
```

Comment:
I agree, We may have a more defined separation between the packages but we need models defined by `Table`, `Column ,`Partition` classes. 
We can either duplicates the attributes to some Glue specific wrappers or create some shared definitions but that may be risky in case of hive/glue api changes

Reply:
This is fine to keep as is. If there is some breakage in future (unlikely) then we can take a look at extracting a wrapper.